### ISSUE TITLE
No Xabber on Google Play?

### ISSUE THREAD

**Issue by bowmanonjupiter2** [redsolution/xabber-android#945]
Hi,

I can not find Xabber on Google Play?

**Comment by hueldoeu** [comment]
you should ask the developer for the beta version

**Comment by tcreek** [comment]
Seems to be abandoned project. Even on F-Droid, no update in over 4 years.

**Comment by hueldoeu** [comment]
i had lots of contact with the developer. a really cool guy. xabber is definitely not abandoned.

**Comment by tcreek** [comment]
There has been no activity on F-Droid over 4 years. Removed from Google store, and nothing going on here in 5 years.

**Comment by hueldoeu** [comment]
he told me once since android v10 it's very difficult to write apps with special permissions for android 10 and above. it's like google reached their goals for a google-only android-smartphone, like apple does to iphone.

**Comment by tcreek** [comment]
Yes, special permissions is an issue, but other projects figured out some workarounds.

**Comment by hueldoeu** [comment]
would you be so kind and contact him?

**Comment by andrewnenakhov** [comment]
Hi. The project is definitely not dead, it's that the Android version is currently on pause, while we're actively building iOS and Web versions of Xabber, as well as Xabber server, which is necessary to make it possible XMPP to work well on those platforms. Unfortunately, we don't have the resources to hire additional developer to work on Android version in addition to all other expenses (if you have such resources, pls contact me). We deprioritized android also because it is the easiest platform to work on, and challenges on iOS and Web are far more difficult to overcome than on Android, so plan is to finish with iOS, then use the experience and protocols developed for iOS and port them to Android. 

Old Xabber for Android was abandoned due to a set of unfortunate circumstances: it had major architecture flaws that we couldn't really fix, and it was prone to have really bad performance issues on high-load accounts. In addition, it had a rather poor build system, so even building it was a problem. So when Google started pestering us with their very vague demands to update privacy policies which we don't completely understand what they want (given the decentralized nature of the app I'm still not sure what checkbox to put when it comes to question regarding content available in the app), etc, so we just decided that we could live without Google Play for the moment. And considering our current country of residence, the lack of exposure is maybe a good thing too.

But the good news is that we're almost done with iOS and Web, in iOS the only thing we still need to do is to finish implementing our new identity verification protocol (this is so good it's nuts!), and do some markup fixing in chat screen and maybe rework the contacts screen design and then release the app. After that, we plan to switch to the development of a new Android version with full force.

**Comment by andrewnenakhov** [comment]
The latest build of old Xabber is available here: https://www.xabber.com/rel/client/android/last/xabber.apk

**Comment by tcreek** [comment]
> Hi. The project is definitely not dead, it's that the Android version is currently on pause, while we're actively building iOS and Web versions of Xabber, as well as Xabber server, which is necessary to make it possible XMPP to work well on those platforms. Unfortunately, we don't have the resources to hire additional developer to work on Android version in addition to all other expenses (if you have such resources, pls contact me). We deprioritized android also because it is the easiest platform to work on, and challenges on iOS and Web are far more difficult to overcome than on Android, so plan is to finish with iOS, then use the experience and protocols developed for iOS and port them to Android.
> 
> Old Xabber for Android was abandoned due to a set of unfortunate circumstances: it had major architecture flaws that we couldn't really fix, and it was prone to have really bad performance issues on high-load accounts. In addition, it had a rather poor build system, so even building it was a problem. So when Google started pestering us with their very vague demands to update privacy policies which we don't completely understand what they want (given the decentralized nature of the app I'm still not sure what checkbox to put when it comes to question regarding content available in the app), etc, so we just decided that we could live without Google Play for the moment. And considering our current country of residence, the lack of exposure is maybe a good thing too.
> 
> But the good news is that we're almost done with iOS and Web, in iOS the only thing we still need to do is to finish implementing our new identity verification protocol (this is so good it's nuts!), and do some markup fixing in chat screen and maybe rework the contacts screen design and then release the app. After that, we plan to switch to the development of a new Android version with full force.

Maybe use XScreenSaver's Privacy Policy?

https://www.jwz.org/xscreensaver/google.html


https://mastodon.social/@jwz/112590330296163479

**Comment by andrewnenakhov** [comment]
The real problem is not the privacy policy text per se, but the compound difficulty of everything i listed above together with dufficulty building the app. So we're just really close to releasing iOS version after which we'll get back to working on a next gen android app with full force and focus.

**Comment by tcreek** [comment]
Thanks for the updates

**Comment by sjehuda** [comment]
> The latest build of old Xabber is available here: https://www.xabber.com/rel/client/android/last/xabber.apk

Would you kindly add that link to the Xabber.com site?

I am working on a [server-side HTML invitation page for XMPP](https://git.xmpp-it.net/sch/FASI), and I want the references of Xabber to be current and dated.

![256-fasi_xabber](https://github.com/user-attachments/assets/256e1a4d-bc51-43db-8cd8-04015fd1ca0c)

---


I think that, it would be good if you establish an F-Droid repository of your own for your builds, as Cheogram and Kaidan did.

You can also host your package at the repository IzzyOnDroid.

**Comment by sjehuda** [comment]
I think that, it would be good if you establish an F-Droid repository of your own for your builds, as Cheogram and Kaidan did.

You can also host your package at the repository IzzyOnDroid.

**Comment by atirage21** [comment]
This uploaded last apk have bug with darj theme. After change settibg to dark mode is broken this application. Also have issue long time also older application, that i am contacted from spam servers every day in hundreds request. In same time i have set request only from my list contact.
